src/test/java/io/xpring/xrpl/XrpClientIntegrationTests.java

```diff
@@ -41,14 +43,22 @@
   private static final String XRPL_ADDRESS = "XVwDxLQ4SN9pEBQagTNHwqpFkPgGppXqrMoTmUcSKdCtcK5";
 
   /**
-   * The seed for a wallet with funds on the XRP Ledger test net.
+   * Drops of XRP to send.
    */
-  private static final String WALLET_SEED = "snYP7oArxKepd3GPDcrjMsJYiJeJB";
+  private static final BigInteger AMOUNT = new BigInteger("1");
 
   /**
-   * Drops of XRP to send.
+   * A Wallet with funds on Testnet.
    */
-  private static final BigInteger AMOUNT = new BigInteger("1");
+  private static Wallet WALLET = null;
+
+  static {
```

Comment:
I wouldn't initialize this in a static initializer if you can avoid it; I'd do it in a test annotated with `@BeforeClass` instead. When things fail on static initializers in Java it can produce ugly `NoClassDefFoundError` issues that are confusing to track down.

Reply:
Great, thanks Ian!  I put the wallet initialization in the same place that I noticed `xrpClient` being initialized (in an @Before annotated test) - let me know if there are issues with that I'm not seeing.